src/Widgets/Tooltip.vala

```diff
@@ -20,7 +20,7 @@
  * Clutter actor to display text in a tooltip-like component.
  */
 public class Gala.Tooltip : Clutter.Actor {
-    private static Clutter.Color text_color;
+    private static Gdk.RGBA text_color;
```

Comment:
Instead of storing the `Gdk.RGBA` colour, we could store the `Clutter.Color` and avoid doing the multiplications in the `draw` method.

Reply:
Good call, cached the math!